.circleci/config.yml

```diff
@@ -217,7 +217,7 @@ jobs:
       - run:
          name: Test that not providing a superuser password causes the container to exit
          command: |
-           if docker run flowminder/flowdb:$SAFE_TAG; then
+           if docker run -e FLOWMACHINE_FLOWDB_PASSWORD=foo -e FLOWAPI_FLOWDB_PASSWORD=foo flowminder/flowdb:$SAFE_TAG; then
```

Comment:
Hmm, does this mean that previously it "failed" (i.e., also exited with status 1) but for the wrong reasons? Can you think of any way we can ensure that this only fails in the case that we actually want to test?